Scripts/AddLabMarketplaceImages/AddMarketplaceImages.ps1

```diff
@@ -0,0 +1,131 @@
+﻿<#
```

Comment:
Since this is similar to `AddAllowedVMSizes.ps1`, please incorporate those comments here as well.

Reply:
good call